common-rendering/src/editorialPalette/index.ts

```diff
@@ -0,0 +1,111 @@
+// ----- Imports ----- //
+
+import { ArticleFormat } from '@guardian/libs';
+
+import { fill } from './fill';
+import { text } from './text';
+import { background } from './background';
+import { border } from './border';
+import { hover } from './hover';
+
+// ----- Types ----- //
+
+type Colour = string;
```

Comment:
This is an example of a type synonym, a way to provide a new name for an existing type. The two are interchangeable and it doesn't provide any additional type safety. I think they're mainly used for semantic purposes to aid the programmer reading through the code - the thinking goes that in this case knowing that these functions provide a colour tells you a little more about them than if you just saw that they return a `string`.

I think whether they help or hinder is up for debate. Here's an example where @adamnfish argues against this pattern (different domain and language but same principle I think): https://github.com/guardian/fontastique/pull/10#discussion_r391586410

I don't have a strong opinion and if you think it would be better to remove them or switch to a template string type then 👍. I think that would be a job for a separate PR (mostly likely a straightforward find-replace) - as Iain mentions this isn't something introduced here, it's an existing pattern.